src/routes/popup/agents/components/LiquidOpsAgentListItem.tsx

```diff
@@ -0,0 +1,105 @@
+import { ListItem, Text } from "@arconnect/components-rebrand";
+import browser from "webextension-polyfill";
+import { Flex } from "~components/common/Flex";
+import LiquidOpsIcon from "url:/assets/ecosystem/liquidops.svg";
+import { SvgImageWithBackground } from "./SvgImage";
+import { useLocation } from "~wallets/router/router.utils";
+import { PopupPaths } from "~wallets/router/popup/popup.routes";
+import { tokenData, type TokenData } from "liquidops";
+import { useGateway } from "../liquidops/utils/hooks/useGateway";
+import { useLOSupplyAPY } from "../liquidops/utils/hooks/useLOSupplyAPY";
+import { type ActiveAgentToken } from "../liquidops/utils/hooks/useAvailableTokens";
+import { useMemo, type ComponentProps } from "react";
+import BigNumber from "bignumber.js";
+import { formatNumber } from "../liquidops/utils/format";
+import { EventType, trackEvent } from "~utils/analytics";
+
+interface Props {
+  activeTokens: ActiveAgentToken[];
+}
+
+export const LiquidOpsAgentListItem = ({ activeTokens }: Props) => {
```

Comment:
Hover styles for the whole Liquid Ops Agent item is confusing. When hovering, the whole item background changes, but the user cannot differentiate when the click is going to go to the  "Show more" or to the children items. See:

![image](https://github.com/user-attachments/assets/d4eac690-d0c4-4f6c-934f-e64be27a4a58)